rca/shortcourses/models.py

```diff
@@ -332,6 +409,15 @@ def clean(self):
             errors["contact_url"].append(
                 "Only one of URL or an Email value is supported here"
             )
+        if (
+            not self.access_planit_course_id
+            and not self.manual_registration_url
+            and self.show_register_link
```

Comment:
**suggestion** (personal): This _might_ be readable with the positive condition first, and the negation grouped:

```
if self.show_register_link and not (
    self.access_planit_course_id and self.manual_registration_url
):
```